adr/formatter.py

```diff
@@ -21,6 +21,7 @@ def __init__(self, table_cls=SingleTable):
         self.table_cls = table_cls
 
     def __call__(self, data):
+        print(data)
```

Comment:
nit: leftover debug statement?

Reply:
You need to add the app in the cli
```
export FLASK_APP=app.py
```
or just execute it using python

Reply:
Yes @ahal, I think I resolved git issues, it was a pain :)

Reply:
Heh, I still get into git trouble myself a lot, pain is a good word :)

It still doesn't seem to work for me, do you run it from the root of the repo or from the `app` directory? I can get the webserver to run from the `app` directory, but then I get the error when I click on one of the links.

Reply:
I run it from app, what error you get?

Reply:
did you run the setup?

Reply:
Hi @ahal, what do you suggest me to fix this, is it because I use py3, or it shouldn't be an issue?

Reply:
Oh I wonder if there was a merge conflict! That seems likely as some changes recently landed that look like they might have cause the error I'm seeing (I pasted it in a comment above).

@nadaa could you try rebasing your branch on top of the latest master and trying again? I bet you'll be able to reproduce it then.

Reply:
@ahal  I got a conflict when I rebase!

Reply:
I made changes and it works, but still I am stuck in a deep conflict!